packages/paste-core/components/code-block/src/CodeBlockTheme.ts

```diff
@@ -0,0 +1,203 @@
+import {css} from '@twilio-paste/styling-library';
+import type {CSSObject} from '@twilio-paste/styling-library';
+import {useTheme} from '@twilio-paste/theme';
+
+export const getPasteSyntaxTheme = (maxLines?: number): CSSObject => {
+  const theme = useTheme();
+
+  return css({
+    'code[class*="language-"]': {
+      color: '#d6deeb',
+      fontFamily: 'fontFamilyCode',
+      textAlign: 'left',
+      whiteSpace: 'pre',
+      wordSpacing: 'normal',
+      wordBreak: 'normal',
+      wordWrap: 'normal',
+      lineHeight: '1.5',
+      fontSize: '1em',
+      MozTabSize: '4',
+      OTabSize: '4',
+      tabSize: '4',
+      WebkitHyphens: 'none',
+      MozHyphens: 'none',
+      msHyphens: 'none',
+      hyphens: 'none',
+    },
+    'pre[class*="language-"]': {
+      color: 'white',
+      fontFamily: 'fontFamilyCode',
+      textAlign: 'left',
+      whiteSpace: 'pre',
+      wordSpacing: 'normal',
+      wordBreak: 'normal',
+      wordWrap: 'normal',
+      lineHeight: '1.5',
+      fontSize: '1em',
+      MozTabSize: '4',
+      OTabSize: '4',
+      tabSize: '4',
```

Comment:
question: what does this translate to?

Reply:
It changes the size of the tab character: https://developer.mozilla.org/en-US/docs/Web/CSS/tab-size